.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+**IMPORTANT: Please make sure you have checked the following list of DOs and DONTs before submitting a pull request.**
+
+*DO*
+
+- If this PR resolves a currently open issue, **refer** to that issue in the description of your PR. For example, if it closes issue #35, say "Resolves issue #35". GitHub will automatically hyperlink the issue and associate the PR to it.
+
+- If this PR introduces a major UI change, include a screenshot showing the specific change in behavior. You can drag and drop a screenshot directly onto the PR text to have it attached by GitHub.
+
+*DO NOT*
+
+- Do not manually change any automatically generated files. In particular, do not change the `*.html`, `*.json` or `*.csv` files. For the majority of functionality you will want to edit the Python script. Be sure that you do not commit any locally changed files when you submit your PR. 
```

Comment:
These should all be in HTML comments (`<!-- -->`) so that they don't show up in the new PR.

I think phrasing it as a checklist (which can be uncommented) would be more friendly.

Here's an example: https://raw.githubusercontent.com/NixOS/nixpkgs/master/.github/PULL_REQUEST_TEMPLATE.md